vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -135,8 +132,7 @@
       kind: Deployment
       state: absent
       namespace: "{{ NAMESPACE }}"
-    environment:
-      KUBECONFIG: "/tmp/target.yaml"
+      kubeconfig: "/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml"
```

Comment:
capital letter for KUBECONFIG

Reply:
I think it is okay because it is not environment variable but the Ansible k8s module parametr.

Reply:
oups, sorry, my bad. you're right